AlphaWallet/TokenScriptClient/Models/AssetAttributeSyntaxValue.swift

```diff
@@ -273,4 +321,48 @@ extension Dictionary where Key == AttributeId, Value == AssetAttributeSyntaxValu
     mutating func setTokenType(string: String) {
         self["tokenType"] = .init(directoryString: string)
     }
-}
\ No newline at end of file
+
+    mutating func setMeltValue(string: String?) {
+        self["meltValue"] = string.flatMap { .init(directoryString: $0) }
+    }
+
+    mutating func setMeltFeeRatio(int: Int?) {
+        self["meltFeeRatio"] = int.flatMap { .init(int: BigInt($0)) }
+    }
+
+    mutating func setMeltFeeMaxRatio(int: Int?) {
+        self["meltFeeMaxRatio"] = int.flatMap { .init(int: BigInt($0)) }
+    }
+
+    mutating func setTotalSupply(string: String?) {
+        self["totalSupply"] = string.flatMap { .init(directoryString: $0) }
+    }
+
+    mutating func setCirculatingSupply(string: String?) {
+        self["circulatingSupply"] = string.flatMap { .init(directoryString: $0) }
+    }
+
+    mutating func setReserve(string: String?) {
+        self["reserve"] = string.flatMap { .init(directoryString: $0) }
+    }
+
+    mutating func setNonFungible(bool: Bool?) {
+        self["nonFungible"] = bool.flatMap { .init(bool: $0) }
+    }
+
+    mutating func setBlockHeight(int: Int?) {
+        self["blockHeight"] = int.flatMap { .init(int: BigInt($0)) }
+    }
+
+    mutating func setMintableSupply(bigInt: BigInt?) {
+        self["mintableSupply"] = bigInt.flatMap { .init(int: $0) }
+    }
+
+    mutating func setTransferable(string: String?) {
```

Comment:
Could you help to move the "setter" functions to below each matching property? Refer to the comment near the top of the extension:

> //"setter" functions are intentionally grouped with their complementary "getter" functions

Easier to find :) But if the arg is optional, we should just use the property instead. The other mutating setter functions were created because they accept a non-optional arg.